pkg/sql/logictest/testdata/logic_test/create_table

```diff
@@ -230,9 +235,10 @@ like_defaults  CREATE TABLE public.like_defaults (
                h INT8 NULL,
                j JSONB NULL,
                k INT8 NULL,
+               t TIMESTAMPTZ NULL ON UPDATE current_timestamp():::TIMESTAMPTZ,
```

Comment:
i'd be fine keeping this one. i vote we just decide now that `INCLUDING DEFAULTS` will also copy `ON UPDATE`. but then one thing to check is that a column with both `DEFAULT` and `ON UPDATE` is copied correctly

Reply:
i've changed this to be for DEFAULTS because ALL is a bitmask meaning including it on every option. i'm a bad proofreader.